core/src/main/java/nu/yona/server/subscriptions/service/UserDto.java

```diff
@@ -223,4 +223,9 @@ public void assertMobileNumberConfirmed()
 			throw MobileNumberConfirmationException.notConfirmed(mobileNumber);
 		}
 	}
+
+	public boolean isChanged(UserDto original)
+	{
+		return !mobileNumber.equals(original.mobileNumber) || privateData.isChanged(original.privateData);
```

Comment:
How about `emailAddress` and `isMobileNumberConfirmed`?

Reply:
See next comment.